### ISSUE TITLE
Using `windows-latest` in benchmarking results in "Error provisioning ressources"

### ISSUE THREAD

**Issue by Marvin-Brouwer** [Marvin-Brouwer/FluentSerializer#244]
**Describe the bug**
When benchmarking in a matrix with `windows-latest` for some reason the pipeline gets cancelled with the following message:
```txt
Benchmark XML serializer / net7.0 windows-latest
An error occurred while provisioning resources (Error Type: Disconnect).
```

**To Reproduce**
1. Add `windows-latest` to the benchmark pipeline matrix
2. Run in GitHub pipelines
3. See the error after some time.

**Expected behavior**
Running the pipeline

**Additional context**
https://github.com/Marvin-Brouwer/FluentSerializer/actions/runs/3775748865

**Comment by Marvin-Brouwer** [comment]
Fixed in #146

**Comment by Marvin-Brouwer** [comment]
Apparently something is starving the resources.
I made a support request to see if there's something we can do about it:
https://github.com/actions/runner-images/issues/6841#issuecomment-1365772518

**Comment by Marvin-Brouwer** [comment]
Fixed in #8
